.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+
+    steps:
```

Comment:
we will need an step to upload the docker image.
It's cool if you want to do that in a separate PR